e2e-tests/IIoTPlatform-E2E-Tests/RegistryHelper.cs

```diff
@@ -70,55 +70,125 @@ public class RegistryHelper : IDisposable {
             }
         }
 
+        /// <summary>
+        /// Wait until one successful deployment is reported.
+        /// </summary>
+        public async Task WaitForSuccessfulDeploymentAsync(
+            Configuration deploymentConfiguration,
+            CancellationToken ct
+        ) {
+
+            try {
+                while (true) {
```

Comment:
>while (true) { [](http://example.com/codeflow?start=16&length=14)

is the cancellation token properly linked& used here? if not,  think we should add a maximum number of retries here to avoid an infinite loop.

Reply:
Yes, it is passed from the test itself and the method does not have a default one, so it should work as expected.